src/librustc_typeck/error_codes.rs

```diff
@@ -3891,6 +3891,25 @@ details.
 [issue #33685]: https://github.com/rust-lang/rust/issues/33685
 "##,
 
+E0587: r##"
+A type has conflicting packed and align representation hints.
+
+Erroneous code example:
+
+```compile_fail,E0587
+#[repr(packed, align(8))] // error!
+struct Umbrella(i32);
+```
+
+You can't use `packed` and `align` hints on a same type. If you want to pack a
+type to a given size, you can provide a size to packed:
+
+```
+#[repr(packed(1))] // ok!
```

Comment:
It feels odd that the example uses `align(8)` above and `packed(1)` here (which is also degenerate because `packed` and `packed(1)` are equivalent).  Is there a way to demonstrate with the same value?